packages/flutter_tools/gradle/src/main/kotlin/DependencyVersionChecker.kt

```diff
@@ -428,6 +407,8 @@ internal class Version(
         return 0
     }
 
+    override fun equals(other: Any?): Boolean = other is Version && compareTo(other) == 0
```

Comment:
This was required so that assertEquals would pass on 2 version instances with the same values.